bitcoind-rpc-test/src/test/scala/org/bitcoins/rpc/v18/BitcoindV18RpcClientTest.scala

```diff
@@ -16,12 +16,9 @@ class BitcoindV18RpcClientTest extends BitcoindRpcTest {
   behavior of "BitcoindV18RpcClient"
 
   it should "be able to start a V18 bitcoind instance" in {
-    for {
-      client.start().flatMap}
-    yield {
-        assert(client.version == BitcoindVersion.V18)
-      }
 
+    client.getNewAddress.flatMap(client.generateToAddress(100, _))
```

Comment:
The client isn't actually started here. If you want to make sure the client is started and then assert on the version, it should look like either: 

```scala
client.start().map { _=> 
  assert(client.version == BitcoindVersion.V18)
}
```

or:
```scala
for {
  _ <- client.start()
} yield assert(client.version == BitcoindVersion.V18
```

Both snippets are equivalent, `for`-expressions are just syntactic sugar for `map`/`flatMap`